int/hwsc-user-svc/test_create.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+docker rm -f devtest;\
+docker run --name devtest --mount type=bind,source="$(pwd)",target=/sql -d postgres:alpine;\
+sleep 5;\
+docker exec -it devtest psql -U postgres -c "CREATE DATABASE testdb;";\
+docker exec -it devtest psql -U postgres -f /sql/S1_CREATE_DOMAINS.sql;\
+docker exec -it devtest psql -U postgres -f /sql/S2_CREATE_USER_TABLE.sql;\
+docker exec -it devtest psql -U postgres -f /sql/S3_CREATE_DOCUMENT_TABLE.sql;\
+docker exec -it devtest psql -U postgres -f /sql/S4_CREATE_SHARED_TABLE.sql;\
```

Comment:
I do not think this script is still needed. The script names are also wrong.

Reply:
Updated the script, this script is to just a fast tester that table creation and inserts work as expected. Should I move it to gitignore?